src/components/ContextBreadcrumbs.tsx

```diff
@@ -118,11 +119,13 @@ const BreadCrumb = React.memo(
           (staticText ? (
             ellipsize(decodeCharacterEntities(value))
           ) : label === HOME_TOKEN ? (
-            <HomeLink color='gray' size={16} />
+            <HomeLink color='gray' size={16} className={css({ position: 'static' })} />
           ) : (
             <Link className='extend-tap-small' simplePath={simplePath} label={label} />
           ))}
-        {!isDeleting && <Superscript simplePath={simplePath} />}
+        {!isDeleting && (
+          <Superscript simplePath={simplePath} css={{ position: 'relative', left: '-2px', top: '-3px' }} />
```

Comment:
Passing an inline array or object literal as a prop results in a different ref every render, preventing the component from being memoized. We need to ensure each prop has a stable object reference if the value does not change.

In this case since the styles are completely static, you can define them in a `const` at the module level. This unfortunately moves them further from where they are used, and forces us to have an extra variable, but there is no other way to stabilize the object reference when passing in the `css` object.

Note that `className` does not have this problem because it is a `string` and strings are always compared by value.